templates/web/base/admin/users/index.html

```diff
@@ -98,7 +98,7 @@
 [% END %]
 
 [% IF NOT ( searched || role_selected )%]
-<h2>[% loc('Add user') %]</h2>
+<h2>[% loc('Add staff user') %]</h2>
```

Comment:
If a superuser is using this page, they can still add a normal user (which I think is fine), but then this title wouldn't entirely be accurate. I guess it probably is worth showing different title in that case, just to be clear.